pkg/provider/aws/sts_test.go

```diff
@@ -302,7 +302,7 @@ func TestGetSignInToken(t *testing.T) {
 			title: "Server returns 200 but bad body format",
 			handler: func(w http.ResponseWriter, r *http.Request) {
 				w.WriteHeader(200)
-				w.Write([]byte("foo"))
+				w.Write([]byte("foo")) //#nosec G104 -- Errors unhandled, just a test
```

Comment:
Can't we just ignore G104 for all tests? Is there a direct to add at the top instead of adding this line in all occurrences?

Reply:
Yes, we can ignore all G104 errors via the golangci-lint configuration file.